### ISSUE TITLE
backdrop-filter blur not working in Safari

### ISSUE THREAD

**Issue by sebastinto** [varabyte/kobweb#491]
Using [`backdropFilter()`](https://github.com/varabyte/kobweb/blob/c7799cc1d8f5c5125af036531fb17cf429c6a334/frontend/kobweb-compose/src/jsMain/kotlin/com/varabyte/kobweb/compose/ui/modifiers/EffectsModifiers.kt#L8) with a [`blur()` CSSFilter](https://github.com/varabyte/kobweb/blob/c7799cc1d8f5c5125af036531fb17cf429c6a334/frontend/compose-html-ext/src/jsMain/kotlin/com/varabyte/kobweb/compose/css/functions/Filters.kt#L11) is not working in Safari.

This is reproducible by navigating to https://bitspittle.dev/blog/2022/kotlin-site in Safari and observing that the header backdrop is translucent instead of blurred.

### Safari
<img width="732" alt="Screenshot 2024-03-31 at 1 42 28 PM" src="https://github.com/varabyte/kobweb/assets/14228120/d7fe5263-e85a-48ac-a0f0-16ccd9547747">

### Chrome
<img width="734" alt="Screenshot 2024-03-31 at 1 43 48 PM" src="https://github.com/varabyte/kobweb/assets/14228120/5e57bbe0-50a4-4267-9593-5931fc7f42bc">
<br/><br/>
As a workaround in my project I'm using this which seems to be working fine:

```kotlin
fun Modifier.backdropFilter(vararg filters: CSSFilter) = styleModifier {
    if (filters.isNotEmpty()) {
        listOf("backdrop-filter", "-webkit-backdrop-filter").forEach {
            property(it, filters.joinToString(" "))
        }
    }
}
```

but since I'm not well-versed in web development, I'm unsure whether this is a proper fix.

* OS: MacOS
* Kobweb libs version: 0.17.1
* Kobweb CLI version: 0.9.15

**Comment by bitspittle** [comment]
BTW meant to thank you for filing this. I'm also glad you found and shared a workaround. I will look into updating the code in Kobweb.

I was trying to avoid the hacky "-webkit-" prefixes that seem to pervade so many projects, but more because I thought that was an indication that the CSS property was still experimental. However, backdrop-filter not working for Safari is kind of a big deal.

**Comment by bitspittle** [comment]
This should be fixed in 0.17.2 when it goes out (aiming for tomorrow, pending some local tests).
